fwl-migration.Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:16.15.0-alpine as build
+
+WORKDIR /opt/app
+
+RUN apk add --no-cache --update bash curl
+
+COPY package.json yarn.lock lerna.json ./
+
+COPY packages packages
+
+RUN yarn install
+
+RUN yarn lerna run --scope @fwl/migration build
+
+FROM node:16.15.0-alpine as deps
+
+WORKDIR /opt/app
+
+COPY package.json yarn.lock lerna.json ./
+
+RUN yarn global add lerna
```

Comment:
Why global? 🤔

Reply:
ah yes, that's a trick I tried that seems to kinda work but not really 😅 

Here's the logic:
I want to install only the packages that are necessary for `migration` but `migration` requires `database` which requires `logging` so I tried to install them with the `lerna bootstrap` command (which does the install for our packages to deal with the`file: ../` versions). 
However, I don't want to install all dev deps to be able to launch lerna, since it would mean installing everything.
That's why, by installing it globally in this docker layer, I can get rid of it when copying the `node_modules` folder in the next step.

It ... sorta works but `lerna bootstrap` is still installing way more stuff than I would like (but there's still a bug with yarn that don't want to install a package that is in the deps if a dev dep needs it ¯\_(ツ)_/¯)

Reply:
Ok, thank you for the detailed answer. 

That triggers my second question that is about the `learna bootstrap` flags line `25`. I don't find them [in the bootstrap doc](https://github.com/lerna/lerna/tree/main/commands/bootstrap#readme). I only find it in [an "old" issue from 2019](https://github.com/lerna/lerna/issues/2345) 🤔

Reply:
Yup, you're right, I didn't search enough and I guess that explains why it does not do what I want

Reply:
It's odd that they don't talk about --scope in the `bootstrap` command but in the main `README` with `command.bootstrap.scope`

Reply:
The doc is weird, I'll grant you that but the logic is that it's not an option but a "filter" and since it's for all commands, they put it there https://github.com/lerna/lerna/tree/main/core/filter-options

Reply:
heyyyy there are the option I removed 😅

Reply:
My bad, it was the `--include` one. :/